File: CSharp/2011 Final Value of Variable After Performing Operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        //Initialize Summation variable.
        int finalValue = 0;

        //Traverse through the operations list.
        foreach (string x in operations) {
            //If current operations contains '+', increment finalValue by 1. Else do the decrement by 1.
            finalValue += x.Contains("+") ? 1 : -1;
        }

        //After traversing is completed, return latest value of summation variable
        return finalValue;
    }
}

class Program
{
    static void Main(string[] args)
    {
        Solution sol = new Solution();
        string[] operations = new string[] {"X++", "++X", "X--"};
        Console.WriteLine(sol.FinalValueAfterOperations(operations));
    }
}

File: 2011 Final Value of Variable After Performing Operations.dart
class Solution {
  int finalValueAfterOperations(List<String> operations) {
    //Initialize Summation variable.
    int finalValue = 0;

    //Traverse through the operations list.
    operations.forEach((x) {
      //If current operations contains '+', increment finalValue by 1. Else do the decrement by 1.
      finalValue += x.contains("+") ? 1 : -1;
    });

    //After traversing is completed, return latest value of summation variable
    return finalValue;
  }
}

void main() {
  Solution sol = Solution();
  print(sol.finalValueAfterOperations(["X++", "++X", "X--"]));
}
